### PR TITLE
2016/1 patron messages

### PR DESCRIPTION
**Pull request by daniel-beck**
Reusing XebiaLabs and CloudBees 2015/Q4 messages.

The [superfrogs](https://twitter.com/jfrog/status/492068563734106112) are a new addition, but haven't provided a logo yet.

**DO NOT MERGE YET**

### CODE REVIEW

**Comment by rtyler** [comment]
these look fine to me fwiw

File: messages.xml [hunk 1]
```diff
@@ -246,4 +246,46 @@
...
+        <link>https://www.cloudbees.com/continuous-delivery/jenkins-docker</link>
+        <blurb>Learn How to Accelerate Your Continuous Delivery Pipelines</blurb>
+        <logo>https://www.cloudbees.com/sites/default/files/cb-logo-clr.r-square78x78.png</logo>
+      </message>
+    </slot>
+
+    <slot weight="1">
+      <contact>samr@jfrog.com</contact>
+      <message>
+        <caption>The Only Enterprise-Ready Docker Repo for Jenkins</caption>
```
Comment (oleg-nenashev) [inline messages.xml]: Looks too offensive IMHO.

File: messages.xml [hunk 2]
```diff
@@ -246,4 +246,46 @@
...
+        <link>https://www.jfrog.com/artifactory-docker</link>
+        <blurb>Join leading software organizations that rely on JFrog Artifactory to Release Fast, or Die. Start a 30-Day Free Trial &gt;&gt;&gt;</blurb>
```
Comment (KostyaSha) [inline messages.xml]: Are end symbols required?

File: messages.xml (hunk 1 above)
Comment (batmat) [inline messages.xml]: +1. Maybe remove the _Only_ word would make a good compromise as just discussed.

**Comment by daniel-beck** [comment]
Updated PR after discussion in project meeting.

Also, we're using the opportunity of JFrog sending the logo via email to set up logo hosting on our own site.

**Comment by batmat** [comment]
LGTM

**Pull request merged**